src/wmnds/components/content-tiles/related-content/related-content.njk

```diff
@@ -0,0 +1,17 @@
+{% macro wmndsContentTileRelatedContent(params) %}
+
+<div class="wmnds-content-tile wmnds-content-tile--related-content">
+    <div class="wmnds-content-tile__title">
+        <h2>Related Content</h2>
```

Comment:
Again, we don't need the title div.
Add the title class to the h2 element itself and remove the div.